src/ol/renderer/maprenderer.js

```diff
@@ -167,8 +167,9 @@ ol.renderer.Map.prototype.forEachFeatureAtCoordinate =
   for (i = numLayers - 1; i >= 0; --i) {
     var layerState = layerStates[i];
     var layer = layerState.layer;
-    if (ol.layer.Layer.visibleAtResolution(layerState, viewResolution) &&
-        layerFilter.call(thisArg2, layer)) {
+    if (!layerState.managed ||
```

Comment:
No I guess I took @elemoine's words in #4143 too literally here: ```PR #3883 made forEachFeatureAtPixel ignore unmanaged layers``` -> wrt layer filtering was meant I guess. I guess I'll need to filter it out myself, but if I'm reading things correctly, I can only do this after @elemoine's changes get in?